weather_mv/README.md

```diff
@@ -325,6 +325,9 @@ _Command options_:
 * `--ee_qps`: Maximum queries per second allowed by EE for your project. Default: 10.
 * `--ee_latency`: The expected latency per requests, in seconds. Default: 0.5.
 * `--ee_max_concurrent`: Maximum concurrent api requests to EE allowed for your project. Default: 10.
+* `--band_names`: A JSON file which contains the band names for the TIFF file.
```

Comment:
Discrepancy in flag name -- it should be `--band_names_mapping`. Please rectify the same at other places of Readme.md as well.

Reply:
Resolved